public/javascript/main.js

```diff
@@ -478,9 +492,61 @@ $(function(){
     }
 
   }
+
+  if (typeof pageInitFunction === 'function') {
+    pageInitFunction();
+  }
 });
 
 function getQueryParam(key) {
   var match = location.search.match(new RegExp("[?&]" + key + "=([^&]+)(&|$)"));
   return match && decodeURIComponent(match[1].replace(/\+/g, " "));
 }
+
+// collapseDisplayedItems:
+// If there are too many items being displayed in the list, hide
+// all but a few, making sure the target element (if there is one)
+// is included in those that are not hidden. Add a button for
+// revealing the hidden items.
+// Note this is used on the download page, where (currently) there
+// may be more than one such list (e.g., one for each legislature).
+
+function collapseDisplayedItems(
+    $ul,               // unordered list elementthat needs to be collapsed
+                       // note: <ul> must have an id
```

Comment:
yes, using the .next() sibling selector instead is a better solution: since the toggle button is being
injected directly above the list it relates to, so can assume that the list is its next sibling (instead of using ID).